src/Calendar/components/TimeGrid/components/Column.js

```diff
@@ -349,6 +352,7 @@ Column.propTypes = {
   columnWidths: COLUMN_WIDTHS_TYPE.isRequired,
   currentTime: MOMENT_TYPE,
   date: PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(Date)]),
+  disableCurrentTimeIndicator: PropTypes.bool,
```

Comment:
this is a nitpick, but wondering based on the usage of the prop if it should be flipped, `showCurrentTimeIndicator` that defaults to `true`? up to your discretion!

Reply:
I prefer that as well. `!disable` is a double negative and hard for me to wrap my head around

Reply:
Yep, we can do this! I based the naming from a pattern I saw in rosetta, like `disableScrollLock`, `disableFocusLock`, etc.

Reply:
Updated in 0a54d5f